### YamlMime:ManagedReference
items:
- uid: System.IdentityModel.RequestFailedException
  id: RequestFailedException
  children:
  - System.IdentityModel.RequestFailedException.#ctor
  - System.IdentityModel.RequestFailedException.#ctor(System.String)
  - System.IdentityModel.RequestFailedException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  - System.IdentityModel.RequestFailedException.#ctor(System.String,System.Exception)
  langs:
  - csharp
  name: RequestFailedException
  nameWithType: RequestFailedException
  fullName: System.IdentityModel.RequestFailedException
  type: Class
  summary: "A exceção que é gerada se a solicitação especificada (primeira) falhou devido a um motivo externo que não pode ser determinado especificamente."
  syntax:
    content: 'public class RequestFailedException : System.IdentityModel.RequestException'
  inheritance:
  - System.Exception
  - System.IdentityModel.RequestException
  implements: []
  inheritedMembers: []
  platform:
  - net462
- uid: System.IdentityModel.RequestFailedException.#ctor
  id: '#ctor'
  parent: System.IdentityModel.RequestFailedException
  langs:
  - csharp
  name: RequestFailedException()
  nameWithType: RequestFailedException.RequestFailedException()
  fullName: System.IdentityModel.RequestFailedException.RequestFailedException()
  type: Constructor
  assemblies:
  - System.IdentityModel
  namespace: System.IdentityModel
  summary: "Inicializa uma nova instância do <xref href=&quot;System.IdentityModel.RequestFailedException&quot;> </xref> classe."
  remarks: "Este construtor inicializa o <xref:SystemException.Message>propriedade da nova instância para uma mensagem fornecida pelo sistema que descreve o erro.</xref:SystemException.Message> Esta mensagem leva em conta a cultura atual do sistema.       A tabela a seguir mostra os valores de propriedade inicial para a nova instância.      | Propriedade | Valor |   |--------------|-----------|   | <xref:System.Exception.InnerException>| `null`.|   | <xref:System.Exception.Message>| A cadeia de caracteres de mensagem de erro localizada. |</xref:System.Exception.Message> </xref:System.Exception.InnerException>"
  syntax:
    content: public RequestFailedException ();
    parameters: []
  overload: System.IdentityModel.RequestFailedException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.IdentityModel.RequestFailedException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: System.IdentityModel.RequestFailedException
  langs:
  - csharp
  name: RequestFailedException(String)
  nameWithType: RequestFailedException.RequestFailedException(String)
  fullName: System.IdentityModel.RequestFailedException.RequestFailedException(String)
  type: Constructor
  assemblies:
  - System.IdentityModel
  namespace: System.IdentityModel
  summary: "Inicializa uma nova instância do <xref href=&quot;System.IdentityModel.RequestFailedException&quot;> </xref> classe com uma mensagem de erro especificada."
  remarks: "Este construtor inicializa o <xref:System.Exception.Message>propriedade da nova exceção usando o `message` parâmetro.</xref:System.Exception.Message> O conteúdo de `message` se destina a ser compreendidos por humanos. O chamador desse construtor é necessário para garantir que essa cadeia de caracteres foi localizada para a cultura atual do sistema.       A tabela a seguir mostra os valores de propriedade inicial para a nova instância.      | Propriedade | Valor |   |--------------|-----------|   | <xref:System.Exception.InnerException>| `null`.|   | <xref:System.Exception.Message>| A cadeia de caracteres de mensagem de erro. |</xref:System.Exception.Message> </xref:System.Exception.InnerException>"
  syntax:
    content: public RequestFailedException (string message);
    parameters:
    - id: message
      type: System.String
      description: "A mensagem de erro que explica o motivo da exceção."
  overload: System.IdentityModel.RequestFailedException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.IdentityModel.RequestFailedException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  id: '#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)'
  parent: System.IdentityModel.RequestFailedException
  langs:
  - csharp
  name: RequestFailedException(SerializationInfo,StreamingContext)
  nameWithType: RequestFailedException.RequestFailedException(SerializationInfo,StreamingContext)
  fullName: System.IdentityModel.RequestFailedException.RequestFailedException(SerializationInfo,StreamingContext)
  type: Constructor
  assemblies:
  - System.IdentityModel
  namespace: System.IdentityModel
  summary: "Inicializa uma nova instância do <xref href=&quot;System.IdentityModel.RequestFailedException&quot;> </xref> classe com os dados serializados."
  remarks: "Este construtor é chamado durante a desserialização para reconstituir o objeto de exceção transmitido em um fluxo."
  syntax:
    content: protected RequestFailedException (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);
    parameters:
    - id: info
      type: System.Runtime.Serialization.SerializationInfo
      description: "Um <xref href=&quot;System.Runtime.Serialization.SerializationInfo&quot;> </xref> objeto que contém os dados do objeto serializado."
    - id: context
      type: System.Runtime.Serialization.StreamingContext
      description: "Um <xref:System.Runtime.Serialization.StreamingContext>objeto que contém as informações contextuais sobre a origem ou destino.</xref:System.Runtime.Serialization.StreamingContext>"
  overload: System.IdentityModel.RequestFailedException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.IdentityModel.RequestFailedException.#ctor(System.String,System.Exception)
  id: '#ctor(System.String,System.Exception)'
  parent: System.IdentityModel.RequestFailedException
  langs:
  - csharp
  name: RequestFailedException(String,Exception)
  nameWithType: RequestFailedException.RequestFailedException(String,Exception)
  fullName: System.IdentityModel.RequestFailedException.RequestFailedException(String,Exception)
  type: Constructor
  assemblies:
  - System.IdentityModel
  namespace: System.IdentityModel
  summary: "Inicializa uma nova instância do <xref href=&quot;System.IdentityModel.RequestFailedException&quot;> </xref> classe com uma mensagem de erro especificada e uma referência à exceção interna que é a causa dessa exceção."
  remarks: "Uma exceção que é lançada como um resultado direto de uma exceção anterior deve incluir uma referência à exceção anterior no <xref:System.Exception.InnerException>propriedade.</xref:System.Exception.InnerException> O <xref:System.Exception.InnerException>propriedade retorna o mesmo valor que é passado para o construtor.</xref:System.Exception.InnerException>       A tabela a seguir mostra os valores de propriedade inicial para a nova instância.      | Propriedade | Valor |   |--------------|-----------|   | <xref:System.Exception.InnerException>| A referência à exceção interna. |   | <xref:System.Exception.Message>| A cadeia de caracteres de mensagem de erro. |</xref:System.Exception.Message> </xref:System.Exception.InnerException>"
  syntax:
    content: public RequestFailedException (string message, Exception innerException);
    parameters:
    - id: message
      type: System.String
      description: "A mensagem de erro que explica o motivo da exceção."
    - id: innerException
      type: System.Exception
      description: "O <xref:System.Exception>que é a causa da exceção atual.</xref:System.Exception> Se o `innerException` parâmetro não é <xref uid=&quot;langword_csharp_null&quot; name=&quot;null&quot; href=&quot;&quot;> </xref>, a exceção atual é gerada em um <xref uid=&quot;langword_csharp_catch&quot; name=&quot;catch&quot; href=&quot;&quot;> </xref> bloco que manipula a exceção interna."
  overload: System.IdentityModel.RequestFailedException.#ctor*
  exceptions: []
  platform:
  - net462
references:
- uid: System.IdentityModel.RequestException
  isExternal: false
  name: System.IdentityModel.RequestException
- uid: System.IdentityModel.RequestFailedException.#ctor
  parent: System.IdentityModel.RequestFailedException
  isExternal: false
  name: RequestFailedException()
  nameWithType: RequestFailedException.RequestFailedException()
  fullName: System.IdentityModel.RequestFailedException.RequestFailedException()
- uid: System.IdentityModel.RequestFailedException.#ctor(System.String)
  parent: System.IdentityModel.RequestFailedException
  isExternal: false
  name: RequestFailedException(String)
  nameWithType: RequestFailedException.RequestFailedException(String)
  fullName: System.IdentityModel.RequestFailedException.RequestFailedException(String)
- uid: System.String
  parent: System
  isExternal: true
  name: String
  nameWithType: String
  fullName: System.String
- uid: System.IdentityModel.RequestFailedException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  parent: System.IdentityModel.RequestFailedException
  isExternal: false
  name: RequestFailedException(SerializationInfo,StreamingContext)
  nameWithType: RequestFailedException.RequestFailedException(SerializationInfo,StreamingContext)
  fullName: System.IdentityModel.RequestFailedException.RequestFailedException(SerializationInfo,StreamingContext)
- uid: System.Runtime.Serialization.SerializationInfo
  parent: System.Runtime.Serialization
  isExternal: false
  name: SerializationInfo
  nameWithType: SerializationInfo
  fullName: System.Runtime.Serialization.SerializationInfo
- uid: System.Runtime.Serialization.StreamingContext
  parent: System.Runtime.Serialization
  isExternal: true
  name: StreamingContext
  nameWithType: StreamingContext
  fullName: System.Runtime.Serialization.StreamingContext
- uid: System.IdentityModel.RequestFailedException.#ctor(System.String,System.Exception)
  parent: System.IdentityModel.RequestFailedException
  isExternal: false
  name: RequestFailedException(String,Exception)
  nameWithType: RequestFailedException.RequestFailedException(String,Exception)
  fullName: System.IdentityModel.RequestFailedException.RequestFailedException(String,Exception)
- uid: System.Exception
  parent: System
  isExternal: true
  name: Exception
  nameWithType: Exception
  fullName: System.Exception
- uid: System.IdentityModel.RequestFailedException.#ctor*
  parent: System.IdentityModel.RequestFailedException
  isExternal: false
  name: RequestFailedException
  nameWithType: RequestFailedException.RequestFailedException
